guides/v2.2/mtf/mtf_installation.md

```diff
@@ -3,6 +3,10 @@ group: functional-testing-framework-guide
 title: Installation of the Functional Testing Framework Entities
 ---
 
+{: .bs-callout-info}
+The Magento Testing Framework (MTF) has been superseded by the Magento Functional Testing Framework ([MFTF][]).
+While the MTF is still functional, all MTF tests are being ported over to the MFTF. We recommend using the MFTF for testing.
```

Comment:
```suggestion
While the MTF is still functional, all MTF tests are ported over to the MFTF. We recommend using the MFTF for testing.
```

Reply:
Keep those "be" verbs outta here!

Reply:
Please make these changes on the other version pages in this PR.

Reply:
Will do.

Reply:
Actually, the are still in the process of porting. It's a work in progress.